quipucords/api/aggregate_report/view.py

```diff
@@ -0,0 +1,19 @@
+"""View for aggregate report."""
+
+from rest_framework.decorators import api_view
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+
+from api import messages
+from api.aggregate_report.model import get_aggregate_report_by_report_id
+from api.common.util import is_int
+
+
+@api_view(["GET"])
+def aggregate_report(request, report_id=None):
+    """Lookup and return a details system report."""
+    if report_id is not None:
+        if not is_int(report_id):
+            error = {"report_id": [messages.COMMON_ID_INV]}
+            raise ValidationError(error)
```

Comment:
I know we have many code snippets like this, but they are useless.
Given how the URL is built, `report_id` will always be present and will always be an `int`. 
If it isn't, this code won't be reached and the user will likely get a 404 response.

IoW, I'd drop this code.